packages/components/src/__future__/Icon/Icon.spec.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+import { render, screen } from "@testing-library/react"
```

Comment:
Aren't we doing these in Storybook these days?

Reply:
Not for DOM-only tests :)